### DISCUSSION TITLE
Page "Request a discount" resets after clicking "Process my application"

### DISCUSSION THREAD

**Discussion by joaofbravo** [community/community#66484] [GitHub Education]
### Hi everyone,

I do not have a date on my student card. What I have is access to the university web portal which contains all this information. I took a print screen which shows the university name and logo, the current date, my photo, name and university email and my enrollment status with the date until when I'm enrolled (31.08.2027).

When I upload the JPG on the student reverification page and click "Process my application", the page just reloads and asks for it again. It does not show any warning like "missing photo" or "missing date", etc.

How can I solve this?

Here is the image without sensitive information:
![IST_student_verification](https://github.com/community/community/assets/30523313/e235b083-ef08-48ad-8113-ecd93a180c51)

**Comment by ShahbazShaddy** [comment]
It seems like you're experiencing a technical issue with the student re-verification page. Here are some steps you can take to try and resolve the problem:

1. **Clear Your Browser Cache**: Sometimes, cached data can cause issues with web pages. Try clearing your browser's cache and cookies and attempt the upload again.

2. **Use a Different Browser or Device**: Sometimes, browser-specific issues can occur. Try using a different browser or device to see if the problem persists.

3. **Check File Format and Size**: Ensure that the JPG file you're uploading meets any specified requirements regarding file format (JPEG) and size limits.

4. **Inspect the Uploaded Image**: After you've uploaded the image, check to make sure that the details you need to verify (university name and logo, current date, photo, name, university email, and enrollment status) are clearly visible.

5. **Contact IT Support**: If the issue persists, it might be a technical problem with the re-verification page itself. Contact your university's IT support or the department responsible for student re-verification for assistance.

6. **Verify Required Fields**: Double-check that you've filled in all required fields on the re-verification page before attempting to upload the image.

7. **Try a Different File**: If you have the option, try saving the image in a different format (e.g., PNG) and attempt to upload it.

8. **Document the Issue**: Take screenshots or record a video of the process, showing the steps you take and any error messages (if any). This can be useful when seeking help from your university's IT support.

9. **Reach Out to Your University's Student Services**: If none of the above steps work, get in touch with your university's student services or registrar's office. They should be able to provide further assistance or alternative methods for re-verification.

Remember to be patient and persistent when dealing with technical issues like this. It might take a bit of troubleshooting to find the solution.

**Reply by joaofbravo** [reply]
I’ve tried the first 4 points. Same result. 

About point 6, there are no required fields in the verification page, it only asks to upload a photo proof.

Here is what happens: I upload the print screen as a JPEG and click Process.
![IMG_0967](https://github.com/community/community/assets/30523313/c776ea8d-111e-4a18-9da0-5ee6570a07d4)

After reaching 100%, it just reload the page without warnings.
![IMG_0968](https://github.com/community/community/assets/30523313/69087f5d-f634-4355-8ce8-a47e2a915aff)

The university teams say they do not deal with the issue at all.
What can I do?

**Reply by Ritik82** [reply]
Same issue.

**Reply by yippp** [reply]
Same issue

**Reply by KadenMc** [reply]
Same issue!

**Reply by dpmacisaac** [reply]
Same Issue! The contact us page is also not letting me put a request in to an actual person. This is frustrating :(
